### DISCUSSION TITLE
Magic link not working - Supabase User Management App, Flutter, iOS

### DISCUSSION THREAD

**Discussion by rogersm-uwosh** [supabase/supabase#21583] [Questions] [answered]
I'm working through the Supabase User Management tutorial / app for Flutter. It's exceptionally well written. When I click on the magic link, however, I get an error message -- that the link is invalid (Safari). Firefox can't find it either.

The link that I was emailed looks like this (except for the ...'s)

https://opcp...cbgi.supabase.co/auth/v1/verify?token=pkce_...6882&type=magiclink&redirect_to=io.supabase.flutterquickstart://login-callback/

I checked my redirect_to, and it matches what's in Supabsae (including the ending /) AFAICT.

Any idea what's wrong, or how to debug this? I would really appreciate some help!

**Comment by dshukertjr** [comment, marked as answer]
I think you are using two devices, initiating the sign-in flow on one device, and opening the email link on another. For the magic link to work, you would need to open the link from the device that you initiated the sign-in flow.
